try {
   
}
catch (e) {
    switch (e) {
        case 'Error':
            
            break;
        default:
            throw e;
    }
}